### PR TITLE
Fix/go mod versioning

### PR DESCRIPTION
**Pull request by repjarms**
### Summary

This PR fixes an issue with the `go.mod` file where we had not specified the `/v2` path segment in the `go.mod`. This caused incompatibilities with installing v2 of the SDK, and caused it to revert to `1.7.0` when calling `go get`.

This PR will release version `2.0.2` of the SDK.

### CODE REVIEW

**Review by mediumTaj — APPROVED** [review]
👍 looks good!

File: .travis.yml
Comment (padamstx) [inline .travis.yml:31]: Because working with the project is a little more tedious with the top level version directory present, I actually added a Makefile to the platform-services-go-sdk project.   Check it out to see if it might be useful for you as well:
https://github.com/IBM/platform-services-go-sdk/blob/master/Makefile
https://github.com/IBM/platform-services-go-sdk/blob/master/.travis.yml

With that sort of setup, I can run the following commands at the project root:
```
make all
make build
make unittest
make tidy
make lint
etc.
```

File: v2/go.sum
Comment (padamstx) [inline v2/go.sum:2]: this doesn't look right to me.  we shouldn't have any code in your project that imports the go core without the "v4" part being included in the import path.

File: v2/go.sum
Comment (padamstx) [inline v2/go.sum:4]: should probably be using the 4.9.4 release of the go core (the latest)

File: v2/go.mod
Comment (padamstx) [inline v2/go.mod:13]: you shouldn't need a "require" entry for your own library.  do you still have some code that uses the OLD 1.7.0 version of the go sdk???

File: v2/go.sum
Comment (padamstx) [inline v2/go.sum:21]: not sure why you have multiple versions of strfmt listed in go.sum.   the ps go-sdk only has this:
```
github.com/go-openapi/strfmt v0.19.10 h1:FEv6Pt/V4wLwP4vOCZbWlpfmi8kj4UiRip34IDE6SGw=
github.com/go-openapi/strfmt v0.19.10/go.mod h1:qBBipho+3EoIqn6YDI+4RnQEtj6jT/IdKm+PAlXxSUc=
```
and the go.mod has an explicit dependency on that package:
```
module github.com/IBM/platform-services-go-sdk

go 1.12

require (
	github.com/IBM/go-sdk-core/v4 v4.9.0
	github.com/go-openapi/strfmt v0.19.10
	github.com/google/uuid v1.1.1
	github.com/joho/godotenv v1.3.0
	github.com/onsi/ginkgo v1.14.2
	github.com/onsi/gomega v1.10.3
	github.com/satori/go.uuid v1.2.0
	github.com/stretchr/testify v1.6.1
)
```

**Review by padamstx — APPROVED** [review]
I left a few comments about dependencies that should probably be resolved before merging and along with a suggestion to add a Makefile to simplify various build/test steps, but otherwise the changes look good.

File: .travis.yml
Comment (repjarms) [reply .travis.yml:31]: @padamstx that's a good point. I will likely add Make and update the configuration in a future PR so that we can prioritize this release to resolve issues with moving to the latest major.

**Comment by repjarms** [comment]
@padamstx I believe I've addressed the issues by regenerating the `go.sum`. I mentioned in the Make comment, but that is something I can look into on a future PR since it doesn't immediately block the release for now.

**Comment by repjarms** [comment]
@padamstx @mediumTaj I updated the PR and did end up including the `Makefile` since that cleaned up a lot of the build process. Also I realized that the reason that the SDK was attempting to include itself as a module was due to the import statements in the generated code not specifying `v2` for the `common` package, which is provided within the SDK. Therefore, the go module resolver was attempting to find the `v1` package, and not finding it locally, retrieved it from the `1.7.0` version of the SDK.

**Pull request merged**
